pyproject.toml

```
@@ -12,7 +12,7 @@ dependencies = [
     "traitlets>=5.0.5",
     "bqplot>=0.12.37",
     "bqplot-image-gl>=1.4.11",
-    "glue-core>=1.17.1",
+    "glue-core>=1.17.1,<1.19.0",
```

If we do this, maybe we don't have to unpin it when/if Tom releases a 1.19.1 with a fix?
```suggestion
    "glue-core>=1.17.1,!=1.19.0",
```

---

Good point. Unless 1.19.1 is also broken 😅